src/main/java/duke/FindCommand.java

```diff
@@ -0,0 +1,36 @@
+package duke;
+
+import java.util.ArrayList;
+import java.io.IOException;
+
```

Comment:
Perhaps you can have a Javadoc comment here briefly describing the class?

Reply:
Alright thanks